ydkgen/api_model.py

```diff
@@ -584,7 +579,6 @@ def stmt(self, stmt):
             if pos_stmt is not None:
                 self._pos_map[bit_stmt.arg] = pos_stmt.arg
 
-
```

Comment:
These lines shouldn't be deleted :) There needs to be a 2 line space between classes according to pep8 standard